packages/batch-submitter/src/batch-submitter/batch-submitter.ts

```diff
@@ -223,6 +241,41 @@ export abstract class BatchSubmitter {
 
     this.logger.info('Received transaction receipt', { receipt })
     this.logger.info(successMessage)
+    this.bsMetrics.gasCostPerSubmission.observe(receipt.gasUsed.toNumber())
     return receipt
   }
+
+  /*********************
+   * Private Functions *
+   ********************/
```

Comment:
Seems unnecessary; we can use a linter if we need to force this kind of thing or we can just see `private` is used a lot; IMO.